UNRELEASED.md

```diff
@@ -1 +1,5 @@
 # Unreleased
+
+**Fixed:**
+
+- Fix calendar year pill the react native bridge
```

Comment:
Could you give some context what was fixed?

Reply:
The year pill doesn't show in the react native otherwise. Why exactly is hard to tell, it probably has to do with how things work over the bridge (js code running in a different thread). We had to do a similar thing in `CaldendarView.requestLayout` and that was based in the react native source code (https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/ReactAndroid/src/main/java/com/facebook/react/views/picker/ReactPicker.java#L75).